ehb_service/apps/api/views/externalrecord.py

```diff
@@ -291,16 +291,13 @@ def post(self, request):
                 "id": label.id,
                 "label": label.label
             })
-        return json.dumps(response)
+        return Response(response)
 
-
-class ExternalRecordRelationResource(Resource):
+@permission_classes((permissions.AllowAny,))
```

Comment:
Removed the @permissionsAllowAny header from all the classes. Created a api/authentication.py file to override Django authentication so that we can continue authenticating using the api tokens that we've created instead of switching over to Django-generated auth-tokens. 803fea7